src/ReactARTFiber.js

```diff
@@ -489,6 +485,10 @@ const ARTRenderer = ReactFiberReconciler({
     // Noop
   },
 
+  getChildHostContext() {
+    return null;
+  },
+
```

Comment:
Ah, I see. NBD. :grin: Alpha-sorting is just nice to have when visually comparing changes across renderers.